src/api/app/views/webui2/webui/project/subprojects.html.haml

```diff
@@ -16,15 +16,15 @@
           %p
             %i This project has no subprojects
         - if User.current.can_modify?(@project)
-          = link_to('#', data: { toggle: 'modal', target: '#add-new-subproject-modal' }, title: 'Add New Subproject', class: 'nav-link') do
+          = link_to('#', data: { toggle: 'modal', target: '#new-project-modal' }, title: 'Add New Subproject', class: 'nav-link') do
```

Comment:
The `title` is not needed here as we already have the text `Add New Subproject` in the link